std/regex/package.d

```diff
@@ -911,20 +851,48 @@ if (isSomeString!R && isRegexFor!(RegEx, R))
     matching scheme to use depends highly on the pattern kind and
     can done automatically on case by case basis.
 
-    Returns: a $(D RegexMatch) object holding engine state after first match.
+    Returns: a $(D RegexMatch) struct holding engine state after first match.
+
+    A RegexMatch is a forward range of captures, lazily evaluated. It offers the
+    following additional members:
+
+    $(UL
+      $(LI pre: shortcut for $(D .front.pre).)
+      $(LI post: shortcut for $(D .front.post).)
+      $(LI hit: shortcut for $(D .front.hit).)
+      $(LI captures: compatibility alias for previous versions of std.regex.)
+    )
+    
+    Each capture struct offers the following methods:
```

Comment:
Instead just document the Captures struct. This description is at best a skeleton